bait-files/src/main/webapp/register.css

```diff
@@ -0,0 +1,43 @@
+  body {
```

Comment:
This file is not needed for this PR right?

Reply:
Removed